templates/company/mortgages/view_content.html.tx

```diff
@@ -22,9 +22,9 @@
             % }
 
             % if ($charge.status == 'outstanding' || $charge.status == 'part-satisfied') {
-                <h2 class="heading-small" align="right">
-                    <a href=<% $c.config.webfiling_login_url %> class="float-right">Satisfy<span class="visually-hidden"> <% if defined $charge.charge_code {$charge.charge_code} else { $charge.classification.description } %></span></a>
-                </h2>
+                <p>
+                    <a href=<% $c.config.webfiling_login_url %>>Satisfy charge <span class="visually-hidden"><% if defined $charge.charge_code {$charge.charge_code} else { $charge.classification.description } %> on the Companies House WebFiling service</span></a>
```

Comment:
The space at the beginning of the hidden section has been moved into the visible section. The visible link shouldn't have a trailing space.

Reply:
Took direction from the story, will change